R/zzz.R

```diff
@@ -7,14 +7,34 @@
 .onLoad <- function(libname, pkgname) {
   packageStartupMessage("Installing Toolchest client... ")
 
+  # check python configuration
+  packageStartupMessage("Checking Python configuration...")
+  python_path <- NULL
+  if (reticulate::virtualenv_exists("r-reticulate")) {
+    reticulate_python_path <- reticulate::virtualenv_python("r-reticulate")
+    # Rebuild r-reticulate environment if built on an incompatible Python version.
+    if (!python_is_compatible(reticulate_python_path)) {
+      packageStartupMessage("Env r-reticulate has an incompatible Python version. Searching for compatible Python...")
+      python_path <- find_compatible_python()
+      packageStartupMessage("Compatible Python found. Env r-reticulate must be rebuilt.")
```

Comment:
Does this print even if a compatible Python package is not found?

Reply:
`find_compatible_python()` throws an error if a compatible package isn't found.